src/Omega_h_mesh.cpp

```diff
@@ -229,7 +231,7 @@ void Mesh::add_tag(Int ent_dim, std::string const& name, Int ncomps,
     TagPtr ptr(tag);
     tags_[ent_dim].push_back(std::move(ptr));
   }
-  OMEGA_H_CHECK(array.size() == nents_[ent_dim] * ncomps);
+  //OMEGA_H_CHECK(array.size() == nents_[ent_dim] * ncomps);
```

Comment:
Please remove commented code.
This seems like a decent check to make.  Why not keep it?

Reply:
That check will not allow storing fields on a subset of mesh entities